doozerlib/rpmcfg.py

```diff
@@ -70,6 +71,10 @@ def clone_source(self):
             out, _ = exectools.cmd_assert(["git", "rev-parse", "HEAD"])
             self.pre_init_sha = source_full_sha = out.strip()
 
+            # gather source repo comit url for audit
+            url, _ = exectools.cmd_assert(["git", "config", "--get", "remote.origin.url"])
+            self.pre_init_commit_url = url.strip().replace(":openshift-priv", "/openshift").replace(".git", "").replace("git@", "https://")
```

Comment:
Using string replacement to convert the URLs is not correct. Please see how we populate container.yaml in during image rebase.